Stack/Opc.Ua.Core/Stack/State/NodeState.cs

```diff
@@ -3669,7 +3669,7 @@ protected virtual ServiceResult ReadNonValueAttribute(
 
                     if (ServiceResult.IsGood(result))
                     {
-                        value = (ushort)accessRestrictions;
+                        value = accessRestrictions.HasValue ? (ushort)accessRestrictions : (object)null;
```

Comment:
doesn`t this throw when accessRestrictions is null?

Reply:
no but I can add the null Check above, just wanted to make the 
intention clear of assinging null to value

Reply:
ok, I wasn't aware of these Nullable extension methods. Apparently check in the if saves an unnecessary assignment, I would prefer that :-)